javascript/javascript1/week2/event.js

```diff
@@ -0,0 +1,10 @@
+//Event application
+const daysFromToday= prompt("How many days from today is the event?");
+const today = prompt("Which day is today?")
+function getEventWeekday(daysFromToday,today){
+    const weekdays = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday",]; 
+    const todayIndex = weekdays.indexOf();
```

Comment:
In this case it will return undefined, since you forgot to add "today" as a parameter for indexOf().
```suggestion
  const todayIndex = weekdays.indexOf(today);
```